src/queue.c

```diff
@@ -6563,6 +6573,14 @@ _dispatch_runloop_queue_handle_init(void *ctxt)
 		}
 	}
 	handle = fd;
+#elif defined(__unix__) && !defined(__linux__)
+	int fds[2];
```

Comment:
Do we not have eventfd or an equivalent on OpenBSD? What about https://man.openbsd.org/event.3?

Reply:
This is an API similar to libevent, and behaves differently from eventfd.

Here, I thought we could use somewhat more standard POSIX-ish API surface that would be more portable (especially when `defined(__unix__)`.)

A more platform-specific API could come later.